pkg/cache/cache_test.go

```diff
@@ -92,7 +93,10 @@ func TestExpireRegionCache(t *testing.T) {
 	re.True(ok)
 	re.Equal(3.0, value)
 
-	re.Equal(2, cache.Len())
```

Comment:
We have slept 20ms, why is it not expired?

Reply:
maybe dogc goroutine did not run in time. I try to run test without this pr. I add a log before `re.Equal` and we can see gc is slow than `re.Equal`
```
[2023/08/31 14:38:07.447 +08:00] [INFO] [cache_test.go:98] [count] [cache-len=3]
[2023/08/31 14:38:07.448 +08:00] [INFO] [ttl.go:165] ["TTL GC items"] [count=1]
    cache_test.go:99: 
        	Error Trace:	/home/lhy1024/pd/pkg/cache/cache_test.go:99
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 3
        	Test:       	TestExpireRegionCache
--- FAIL: TestExpireRegionCache (0.07s)
FAIL
FAIL	github.com/tikv/pd/pkg/cache	0.075s
FAIL
```